tests/protos/test_fully_connected.py

```diff
@@ -22,19 +23,30 @@ def fully_connecteds(
     st.SearchStrategy[Tuple[fully_connected_pb2.FullyConnected, Dict]],
 ]:
     """Returns a SearchStrategy for a FC layer plus maybe the kwargs."""
-    kwargs = {}
+    eps = 1e-8
 
+    kwargs = {}
+    to_ignore = []
     kwargs["num_hidden_layers"] = draw(st.integers(0, 3))
     if valid_only and kwargs["num_hidden_layers"] == 0:
         kwargs["hidden_size"] = None
         kwargs["activation"] = activation_pb2.Activation(
             identity=empty_pb2.Empty()
         )
+        to_ignore.append("dropout")
     else:
         kwargs["hidden_size"] = draw(st.integers(1, 32))
         kwargs["activation"] = draw(activations())
-
-    all_fields_set(fully_connected_pb2.FullyConnected, kwargs)
+        if draw(st.booleans()):
```

Comment:
Neater way for this that also shrinks towards `None` and avoids using `to_ignore`: 

```suggestion
        kwargs["dropout"] = st.one_of(st.none(), st.floats(min_value=0.0, max_value=1.0))
```

Reply:
Done. Added an extra `draw(..) to suggested change